mod-source-record-manager-server/src/test/java/org/folio/rest/impl/metadataProvider/MetadataProviderJobExecutionAPITest.java

```diff
@@ -547,6 +547,21 @@ public void shouldReturnFilteredCollectionByHrIdOnGet() {
       .body("jobExecutions[0].hrId", is(expectedHrid));
   }
 
+  @Test
+  public void shouldReturnFilteredCollectionByExcludeJobProfileNameOnGet() {
+    constructAndPostInitJobExecutionRqDto(5);
+
+    RestAssured.given()
+      .spec(spec)
+      .when()
+      .queryParam("excludeJobProfileNames", "importBib2.bib")
```

Comment:
please add test to exclude several jobs that have similar names satisfying filter with wildcard